doc/languages-frameworks/qt.section.md

```diff
@@ -23,10 +23,12 @@ stdenv.mkDerivation {
 }
 ```
 
+The same goes for Qt5 where libraries and tools are under `libsForQt5`.
```

Comment:
Generally, "Qt 5", "Qt 6".

Reply:
I'm also confused with this. Looks like some libraries aren't exposed in `qt5` set? For example, `libsForQt5.qzxing` exists, but `qt5.qzxing` doesn't.

Reply:
This is not about the package set, this is about the wording. Upstream uses "Qt 5" and "Qt 6" to refer to versions, with a space.

Reply:
Oh, my mistake